pyproject.toml

```diff
@@ -91,7 +94,8 @@ docs = [
     "myst-parser >=0.13",
     "pathspec >=0.10.1",
     "pyproject-metadata >=0.5",
-    "sphinx >=7.0,<=7.1.2",  # workaround bug in setuptools, fix in pypa/setuptools#4023
+    "setuptools",
```

Comment:
Why is `setuptools` still needed here? I thought only `sphinx-bibtex` is affected by this.

Reply:
I think this was because we have a setuptools plugin, and it shows up in the docs. I'll probably move it to it's own repo in a couple of releases.